docs/howto/object-storage/swift/private-container.md

```diff
@@ -0,0 +1,171 @@
+# Working with a private Swift container
+
+## Prerequisites
+
+In order to create a Swift container, be sure that you have
+[installed and configured](../) the required command-line interface
+(CLI) tools.
+
+
+## Creating a private container
+
+To create a private container (that is, one that can only be accessed
+with proper Swift API credentials), use the following command:
+
+=== "OpenStack CLI"
+    ```console
+    $ openstack container create private-container
+    +---------------------------------------+-------------------+----------------------------------------------------+
+    | account                               | container         | x-trans-id                                         |
+    +---------------------------------------+-------------------+----------------------------------------------------+
+    | AUTH_30a7768a0ffc40359d6110f21a6e7d88 | private-container | tx00000ddb0f9e2a50ad881-00638dbf9c-300de11-default |
+    +---------------------------------------+-------------------+----------------------------------------------------+
+    ```
+=== "Swift CLI"
+    ```console
+    $ swift post private-container
+    ```
+    This command produces no output.
+
+
+## Retrieving container information
+
+To create a list of all containers accessible with your current set of
+credentials, use this command:
+
+=== "OpenStack CLI"
+    ```console
+    $ openstack container list
+    +-------------------+
+    | Name              |
+    +-------------------+
+    | private-container |
+    +-------------------+
+    ```
+=== "Swift CLI"
+    ```console
+    $ swift list
+    private-container
+    ```
+
+To retrieve more detailed information about an individual container,
+you can also use this command:
+
+=== "OpenStack CLI"
+    ```console
+    $ openstack container show test-container
```

Comment:
I believe this should be `private-container`, right?

Reply:
Indeed it should! Good catch; fixed now.